api/src/wfl/module/xx.clj

```diff
@@ -27,43 +27,33 @@
-     :contamination_sites_mu     (hg38 "contamination-resources/1000g/1000g.phase3.100k.b38.vcf.gz.dat.mu")
-     :contamination_sites_ud     (hg38 "contamination-resources/1000g/1000g.phase3.100k.b38.vcf.gz.dat.UD")
-     :dbsnp_vcf                  (hg38 "Homo_sapiens_assembly38.dbsnp138.vcf")
-     :dbsnp_vcf_index            (hg38 "Homo_sapiens_assembly38.dbsnp138.vcf.idx")
-     :evaluation_interval_list   (hg38 "exome_evaluation_regions.v1.interval_list")
-     :haplotype_database_file    (hg38 "Homo_sapiens_assembly38.haplotype_database.txt")
-     :known_indels_sites_vcfs    [(hg38 "Mills_and_1000G_gold_standard.indels.hg38.vcf.gz")
-                                  (hg38 "Homo_sapiens_assembly38.known_indels.vcf.gz")]
-     :known_indels_sites_indices [(hg38 "Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi")
-                                  (hg38 "Homo_sapiens_assembly38.known_indels.vcf.gz.tbi")]
-     :reference_fasta            (references/reference_fasta)}))
-
-(def ^:private scatter_settings-defaults
-  {:haplotype_scatter_count     50
-   :break_bands_at_multiples_of 0})
-
-(def ^:private papi_settings-defaults
-  {:agg_preemptible_tries 3
-   :preemptible_tries     3})
+  (let [hg38 "gs://gcp-public-data--broad-references/hg38/v0/"
+        hsa  "Homo_sapiens_assembly38"]
+    (merge references/hg38-exome-references
+      references/contamination-sites
+      references/exome-scatter-settings
+      {:calling_interval_list (str hg38 "exome_calling_regions.v1.interval_list")
+       :haplotype_database_file (str hg38 hsa ".haplotype_database.txt")})))
 
 (def ^:private workflow-defaults
-  (let [hg38 (partial str "gs://gcp-public-data--broad-references/hg38/v0/")]
+  (let [hg38          "gs://gcp-public-data--broad-references/hg38/v0/"
+        hsa           "Homo_sapiens_assembly38"
+        bait_set_name "whole_exome_illumina_coding_v1"
+        iv1           (str "HybSelOligos/" bait_set_name "/" "." hsa)]
     {:unmapped_bam_suffix  ".unmapped.bam"
-     :cram_ref_fasta       (hg38 "Homo_sapiens_assembly38.fasta")
-     :cram_ref_fasta_index (hg38 "Homo_sapiens_assembly38.fasta.fai")
-     :bait_set_name        "whole_exome_illumina_coding_v1"
-     :bait_interval_list   (hg38 "HybSelOligos/whole_exome_illumina_coding_v1/whole_exome_illumina_coding_v1.Homo_sapiens_assembly38.baits.interval_list")
-     :target_interval_list (hg38 "HybSelOligos/whole_exome_illumina_coding_v1/whole_exome_illumina_coding_v1.Homo_sapiens_assembly38.targets.interval_list")
+     :cram_ref_fasta       (str hg38 hsa ".fasta")
+     :cram_ref_fasta_index (str hg38 hsa ".fasta.fai")
+     :bait_set_name        bait_set_name
+     :bait_interval_list   (str hg38 iv1 ".baits.interval_list")
+     :target_interval_list (str hg38 iv1 ".targets.interval_list")
      :references           references-defaults
-     :scatter_settings     scatter_settings-defaults
-     :papi_settings        papi_settings-defaults}))
```

Comment:
These are required inputs to the wdl. They're not part of the "references" structure. Why have you removed this?

Reply:
I moved them back.  (Just trying to make sense of stuff.)